src/utils/api.js

```diff
@@ -12,19 +11,24 @@ export const getCategories =()=>{
         return res.data
     })
 }
-export const getReviews = (category)=>{
-    
+export const getReviews = (category,sort_by, order)=>{
+    let queryString= '/reviews'
+    if (sort_by){
+        queryString+=`?sort_by=${sort_by}`
+    }
+    if (order){
+        queryString +=`&order=${order}`
+    }
     if (category){
-        return reviewApi.get(`/reviews?category=${category}`)
-        .then ((res)=>{
-            return res.data
-        })
+        queryString +=`&category=${category}`
     }
-    return reviewApi.get("/reviews", {params:{category:category}})
-    .then ((res)=>{
-        return res.data
+    
+    return reviewApi.get(queryString)
```

Comment:
While this works, using axios params may mean you don't need any if statement logic.